projects/material-addons/src/lib/alert/alert-configuration.ts

```diff
@@ -0,0 +1,19 @@
+import { Inject, InjectionToken, Optional, SkipSelf } from '@angular/core';
+import { AlertType, AlertSize } from './alert.component';
+
+export interface AlertConfiguration {
+  type?: AlertType;
+  size?: AlertSize;
+}
+
+export const MAD_ALERT_CONFIGURATION = new InjectionToken<AlertConfiguration>('mad-alert-configuration');
+
+export const MAD_ALERT_CONFIGURATION_PROVIDER = {
+  provide: MAD_ALERT_CONFIGURATION,
+  deps: [[new Optional(), new SkipSelf(), new Inject(MAD_ALERT_CONFIGURATION)]],
```

Comment:
the factory should not have any dependencies as this should be the MAD default options (so the default default if you will)

Reply:
done